example_data/sampleFHIR_V2.json

```diff
@@ -0,0 +1,1251 @@
+              "coding": [
+                {
+                  "system": "http://hl7.org/variant",
+                  "code": "confirmed-mutation",
+                  "display": true
+                }
+              ]
+            }
+          }
+        ]
+      },
+      "request": {
+        "method": "POST",
+        "url": "Observation"
+      }
+    },
+    {
+      "fullUrl": "Task/5f3f5638-3870-1a14-b490-b6081dfc8352",
+      "resource": {
+        "resourceType": "Task",
+        "id": "5f3f5638-3870-1a14-b490-b6081dfc8352",
+        "meta": {
+          "profile": [
+            "http://hl7.org/fhir/StructureDefinition/task-rec-followup"
+          ]
+        },
+        "status": "requested",
+        "intent": "plan",
+        "code": {
+          "coding": [
+            {
+              "system": "http://loinc.org",
+              "code": "LA14020-4",
+              "display": "Genetic counceling recommended"
+            }
+          ]
+        },
+        "description": "Testing of General Manager's parents is recommended to determine whether the SCN1A c.2101C>T pathogenic variant has arisen de novo and",
+        "for": {
+          "type": "Patient",
+          "reference": "Patient/37176e84-d977-e993-3c49-d76fcfc6e625"
+        }
+      },
+      "request": {
+        "method": "POST",
+        "url": "Task"
+      }
+    },
+    {
+      "fullUrl": "Observation/6d16ee18-5521-16dd-2ba4-b180cb69ca38",
+      "resource": {
+        "resourceType": "Observation",
+        "id": "6d16ee18-5521-16dd-2ba4-b180cb69ca38",
+        "status": "final",
+        "meta": {
+          "profile": [
+            "http://hl7.org/fhir/uv/genomic-reporting/overall-interpretation"
+          ]
+        },
+        "code": {
+          "coding": [
+            {
+              "system": "http://loinc.org",
+              "code": "51968-6",
+              "display": "Genetic disease analysis overall interpretation"
+            }
+          ]
+        },
+        "subject": {
+          "type": "Patient",
+          "reference": "Patient/37176e84-d977-e993-3c49-d76fcfc6e625"
+        },
+        "specimen": {
+          "type": "Specimen",
+          "reference": "Specimen/fb82860d-eabc-a8d0-b341-facdff0ac0f1"
+        },
+        "performer": [
+          {
+            "type": "Practitioner",
+            "reference": "Practitioner/d3447490-96fd-35d0-adf2-0806e5214606"
+          }
+        ],
+        "category": [
+          {
+            "coding": [
+              {
+                "system": "http://terminology.hl7.org/obs-category",
+                "code": "laboratory",
+                "display": "Laboratory"
+              }
+            ]
+          }
+        ],
+        "effectiveDateTime": "2020-06-03",
+        "valueString": "Next generation sequence analysis indicates that General Manager is heterozygous for the MECP2 c.953G>C p.(Arg318Pro) likely pathogenic variant that has been confirmed by Sanger sequence analysis"
```

Comment:
have removed interpretation coded value, only free text remaining

Reply:
Having a look here, we did previously also have a clinical conclusion in addition to this data. e.g. `Confirms the diagnosis of Childhood-nset epileptic encephalopathy (EEOC).`, are we removing this altogether?

Reply:
actually can see that it was duplicated, and this is contained in the diagnostic report conclusion! Will resolve this

Reply:
Yes, pls remove the one in overall interpretation (Observation resource). It's a mistake